install_config/install/docker_registry.adoc

```diff
@@ -553,14 +553,25 @@ Digest: sha256:3662dd821983bc4326bee12caec61367e7fb6f6a3ee547cbaff98f77403cab55
 ----
 ====
 
-== Advanced Configuration
+[[advanced-overriding-the-registry-configuration]]
+== Advanced: Overriding the Registry Configuration
 
-To enable managing the registry configuration file directly it is recommended that the configuration file be mounted as a secret volume.
+You can override the integrated registry's default configuration, found by
+default at *_/config.yml_* in a running registry's container, with your own
+custom configuration. See the official Docker documentation's
```

Comment:
I would like to get these options into our official, supported documentation. Until then can we refer to it as the "upstream registry documentation"?